python/setup.py

```diff
@@ -87,6 +87,10 @@ def find_pybind_headers_dir():
 if len([fn for fn in os.listdir(NGRAPH_CPP_LIBRARY_DIR) if re.search('ngraphd', fn)]):
     NGRAPH_CPP_LIBRARY_NAME = 'ngraphd'
 
+ONNX_IMPORTER_CPP_LIBRARY_NAME = 'onnx_importer'
+if len([fn for fn in os.listdir(NGRAPH_CPP_LIBRARY_DIR) if re.search('onnx_importerd', fn)]):
```

Comment:
@tomdol @tsocha @mbencer  Do we need onnx_importer dependency for the Python API? As we discussed user can use OpenVINO Reader API to load ONNX model and this API can be hided from Python.

Reply:
The onnx_importer is still exposed in `ngraph.impl` python module. We can probably remove it but let's discuss it offline and create a separate story for that.

Reply:
We need the ONNX importer API for testing the ONNX spec.

Reply:
Good! Thank you for the clarification.